src/index.js

```diff
@@ -125,6 +126,41 @@ class Mongo {
     }
   }
 
+  async generateVerificationEmailToken(userId: string, email: string): Promise<void> {
```

Comment:
`email` should be an unverified email or throw if the user doesn't have an unverified email. This should be checked on Accounts server or here?